docs/data-sources/vpcep_endpoints.md

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "VPC Endpoint (VPCEP)"
+---
+
+# flexibleengine\_vpcep\_endpoints
+
+Use this data source to get VPC endpoints.
+
+## Example Usage
+
+```hcl
+data "flexibleengine_vpcep_endpoints" "all_services" {
+}
+
+data "flexibleengine_vpcep_endpoints" "dns_service" {
+  endpoint_service_name = "dns"
+}
+```
+
+## Argument Reference
+
+* `endpoint_service_name` - (Optional, String) Specifies the name of the VPC endpoint service.
+    The value is not case-sensitive and supports fuzzy match.
+
+* `endpoint_id` - (Optional, String) Specifies the unique ID of the VPC endpoint.
+
+* `vpc_id` - (Optional, String) Specifies the unique ID of the vpc holding the VPC endpoint service.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - Specifies a data source ID in UUID format.
+
+* `endpoints` - Indicates the public VPC endpoints information. Structure is documented below.
+
+The `endpoints` block contains:
+
+* `id` - The unique ID of the public VPC endpoint service.
+* `status` - The connection status of the VPC endpoint.
+* `active_status` - The domain status.
+* `endpoint_service_name` - The name of the VPC endpoint service.
+* `endpoint_service_type` - The type of the VPC endpoint.
+* `endpoint_service_id` - The ID of the VPC endpoint service.
+* `vpc_id` - The ID of the VPC holding the VPC endpoint service.
+* `subnet_id` - The ID of the subnet holding the VPC endpoint.
+* `ip` - The IP of the VPC endpoint.
+* `marker_id` - The marker id of the VPC endpoint.
```

Comment:
I think some attributes are attributes empty or meaningless, how about refer to the attributes 
 of `flexibleengine_vpcep_endpoint` resource?

Reply:
done